### PR TITLE
MTL-1462 Remove the SDU partition

### PR DESCRIPTION
**Pull request by rustydb**
Fixes MTL-1462 

Renames SDU to S3FS.

### CODE REVIEW

File: boxes/ncn-node-images/k8s/files/resources/metal/cloud.cfg.d/02_metalmounts.cfg [hunk 1]
```diff
@@ -1,3 +1,3 @@
 #cloud-config
 mounts:
-    - [ LABEL=CRAYSDU, /var/lib/sdu, ext4, "defaults,nofail" ]
+    - [ LABEL=CRAYS3FS, /var/lib/s3fs, ext4, "defaults,nofail" ]
```
Comment (bklei) [inline boxes/ncn-node-images/k8s/files/resources/metal/cloud.cfg.d/02_metalmounts.cfg:3]: not a big deal if too late -- but later yesterday we were thinking the dir might be better named '/var/lib/s3fs_cache' ?  but will have my pr match whatever merges here...

**Review by bklei — APPROVED** [review]

File: boxes/ncn-node-images/k8s/files/resources/metal/cloud.cfg.d/02_metalmounts.cfg (hunk 1 above)
Comment (rustydb) [reply boxes/ncn-node-images/k8s/files/resources/metal/cloud.cfg.d/02_metalmounts.cfg:3]: Nice catch.
I admit I went to our original message this morning and forgot about that detail.

Comment (rustydb) [reply boxes/ncn-node-images/k8s/files/resources/metal/cloud.cfg.d/02_metalmounts.cfg]: Fixed.

**Review by bklei — APPROVED** [review]

**Review by jacobsalmela — APPROVED** [review]

**Review by heemstra — APPROVED** [review]

**Pull request merged**